backend/src/controllers/serviceControllers.js

```diff
@@ -0,0 +1,46 @@
+import githubService from '../thirdParties/github/githubService';
+
+const getAppropriateService = serviceName => {
+  if (serviceName === 'github') {
+    return githubService;
+  }
+  return githubService;
+};
+
+const getServices = async (req, res) => {
+  try {
+    const services = ['github'];
+    res.send({ services });
+  } catch (error) {
+    res.status(500).send();
+  }
+};
+
+// only supporting github for now
+const getActionsForService = async (req, res) => {
+  try {
+    const service = getAppropriateService(req.param('service'));
+    const { actions } = service;
+    const actionNames = actions.map(action => action.name);
+    res.send({ actions: actionNames });
+  } catch (error) {
+    res.status(500).send();
+  }
+};
+
+const getTriggerForService = async (req, res) => {
+  try {
+    const service = getAppropriateService(req.param('service'));
```

Comment:
`req.param('service')` are you calling a function here?

Reply:
Yes, this is a function. Nut now I re-read the express docs, I can use this using `req.params.service` which would be a better choice, I think.

Reply:
yes, please change accordingly